testMVVM/EpisodeCreateCoordinator.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  EpisodeCreateCoordinator.swift
+//  MVVMTestProject
+//
+//  Created by Petr Zvoníček on 25.09.16.
+//  Copyright © 2016 Martin Pinka. All rights reserved.
+//
+
+import UIKit
+
+class EpisodeCreateCoordinator: Coordinator {
+
+    let navigationController: UINavigationController
+    let viewModel: SeasonDetailViewModel
+
+    let wrapperNavigationController: UINavigationController?
+    var viewController: EpisodeCreateViewController
+
+    init(navigationController: UINavigationController, wrapperNavigationController: UINavigationController, viewController: EpisodeCreateViewController, viewModel: SeasonDetailViewModel) {
```

Comment:
I got back to that and you're right that this optional wrapper is not necessary. But it was actually intentional 😉 I wanted to show how to support both modal and non-modal cases. In non-modal case this `wrapperNavigationController` would actually be `nil` and coordinator would know it should push the controller to navigation stack.